test/integration/wallet.test.js

```diff
@@ -228,6 +228,14 @@ describe('@integration: Wallet', () => {
       step(`Then I should see text ${successMessage} in "result box message" element`, () => helper.haveTextOf('.result-box-message', successMessage));
     });
 
+    describe('Scenario: should display the request LSK component when clicking on the tab', () => {
+      step('Given I\'m on "wallet" as "genesis" account', () => setupStep('genesis'));
+      step('And I click "request tab"', () => { helper.clickOnElement('.request-tab'); });
+      step('Then I should see the QR code', () => helper.haveLengthOf('.request-qr-code', 1));
+      step('And I click "send tab"', () => { helper.clickOnElement('.send-tab'); });
```

Comment:
If we want to strictly follow Given-When-Then pattern, then both `And I click` (line 23 and 235) should be `When I click`. `And` is used only when there are two `When`s in a row.